manifests/config.pp

```diff
@@ -55,4 +55,11 @@
     content => $server_config_real,
   }
 
+  if $::osfamily == 'RedHat' and ($version == 'present' or versioncmp($version, '5.9') < 0) {
+    file { '/usr/share/activemq/activemq-data':
+      ensure => link,
+      target => '/usr/share/activemq/data', 
```

Comment:
Should the target here be /var/cache/activemq/data?  (which is where /usr/share/activmeq/data links to)